### DISCUSSION TITLE
Proposal: delay 1.26.0

### DISCUSSION THREAD

**Discussion by haircommander** [cri-o/cri-o#6436] [General]
Typically, the CRI-O community tries to release a corresponding CRI-O version three days after the kube version. While we typically actually end up releasing a week after, it's usually in the ballpark.

However, this release we're plagued with some CI failures. The fix is in the works, and we're investigating workarounds, but until we find something, there's a number of  backlogged PRs that I would like in 1.26.0. I am opening this to suggest to the community (and ask for feedback) as to whether it makes sense to delay the release by a week or two so we can get things sorted out.

**Comment by adrianreber** [comment]
As I have a change that I would like to see in 1.26.0 regarding checkpointing and which is stuck because of CI I am in favour of your proposal.

**Comment by saschagrunert** [comment]
No objections from my side, while assuming that we still plan to release this year.
